### PR TITLE
New package: runit-swap-1.0.0

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/runit-swap/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://github.com/thypon/runit-swap"
+
+do_fetch() {
+  local url="git://github.com/thypon/runit-swap"
+  msg_normal "Fetching source from $url ...\n"
+  git clone --branch v${version} ${url} ${pkgname}-${version}
+}
+
+do_install() {
+  vinstall runit-swap.sh      744 usr/sbin
```
Comment (xtraeme) [inline srcpkgs/runit-swap/template]: That files should be named **runit-swap** and perms **755**.

File: srcpkgs/runit-swap/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  vconf    runit-swap.conf
```
Comment (xtraeme) [inline srcpkgs/runit-swap/template]: You are missing `conf_files="/etc/runit-swap.conf"`.

File: srcpkgs/runit-swap/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  vinstall swap/run           744 etc/sv/swap
```
Comment (xtraeme) [inline srcpkgs/runit-swap/template]: The runit service should be named **runit-swap** not **swap**.

Also perms should be **755** and you should create a **supervise** symlink.

File: srcpkgs/runit-swap/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  vinstall swap/finish        744 etc/sv/swap
+  vinstall 90-runit-swap.conf 644 etc/modprobe.d/
```
Comment (xtraeme) [inline srcpkgs/runit-swap/template]: `/etc/modprobe.d` -> `/usr/lib/modprobe.d`

File: srcpkgs/runit-swap/template [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  local url="git://github.com/thypon/runit-swap"
+  msg_normal "Fetching source from $url ...\n"
+  git clone --branch v${version} ${url} ${pkgname}-${version}
+}
+
+do_install() {
+  vinstall runit-swap.sh      744 usr/sbin
+  vconf    runit-swap.conf
+  vinstall swap/run           744 etc/sv/swap
+  vinstall swap/finish        744 etc/sv/swap
```
Comment (xtraeme) [inline srcpkgs/runit-swap/template]: perms **755**

File: srcpkgs/runit-swap/template (hunk 3 above)
Comment (thypon) [inline srcpkgs/runit-swap/template]: I found in docomunetation the vsv() directives but it says the sv scripts need to be in FILESDIR. Have i to change FILESDIR or could i skip it.

**Comment by xtraeme** [comment]
You did not fix the runit `supervise` symlink issue yet, i.e as you are not using `vsv`, you should provide the `supervise` symlink.

**Comment by xtraeme** [comment]
Please fix the travis CI warnings.

**Comment by xtraeme** [comment]
Also the `if statement` about the `log` service is useless...

**Comment by xtraeme** [comment]
You are missing `noarch=yes`.

**Comment by thypon** [comment]
Maybe i full rebase all the commits?

**Pull request merged**